src/app/(auth)/login/(components)/LoginHunter/LoginHunter.tsx

```diff
@@ -0,0 +1,47 @@
+'use client'
+
+import { useState } from 'react'
+
+// components
+import Box from '@/components/Box/Box'
+import { Button } from '@/components/Button/Button'
+import CheckboxInput from '@/components/Checkbox/Checkbox'
+import TextInput from '@/components/TextInput/TextInput'
+
+const LoginHunter = () => {
+  const [isChecked, setIsChecked] = useState(false)
+  return (
+    <Box>
+      <h2>Login as talent Hunter</h2>
+      <p>
+        Join and login alaways with magic link. Hunters don't have public
+        profiles.
+      </p>
+      <form>
+        <TextInput
+          label="Email"
+          tooltipText=" Lorem ipsum dolor sit amet, consectetur adipisicing elit."
+          addImportantIcon={true}
+          placeholder="eg. peter.parker@oscorp.com"
+          onChange={() => {
+            console.log('')
+          }}
+        />
+        <div style={{ marginBlock: '24px' }}>
+          <CheckboxInput
+            id={'terms'}
```

Comment:
**nit:** Let's move duplicated keys strings into some enum or const like we have in every form currently